infrastructure/helm-chart/templates/ingress.yaml

```diff
@@ -92,13 +92,34 @@ spec:
                 name: api-communication
                 port:
                   number: 80
+          - path: /login
+            pathType: Prefix
+            backend:
+              service:
+                name: api-admin
```

Comment:
The login flow needs to be completed against a single pod. On success, we exchange the OAuth authentication for an Airy session and it becomes statelessly available for all apps.